OpenNefia.Content/CharaMake/CharaMakeAttributeRerollLayer.cs

```diff
@@ -168,10 +172,35 @@ public override void Initialize(CharaMakeData args)
             IsInitialized = true;
         }
 
-        private void Reroll()
+        private void Reroll(bool playSound = true)
         {
-            Sounds.Play(Protos.Sound.Dice);
-            //TODO add rerolling
+            if (playSound)
+                Sounds.Play(Protos.Sound.Dice);
+
+            if (!Data.TryGetValue("race", out RacePrototype race))
```

Comment:
Should put the key name as a `public const string` in the appropriate charamake step.

Reply:
this is accessed from a different layer, it'd need to be static for that. no real way to get a reference here.

Reply:
done in 1839a4d